packages/augur-ui/src/utils/get-odds.ts

```diff
@@ -47,29 +49,38 @@ export const test = (examples = mockExamples) => {
   examples.forEach(example => console.log(convertToOdds({ ...example })));
 };
 
-interface ConvertToOddsType {
+interface ConvertToNormalizedPriceType {
   price: number | string | BigNumber;
   min?: number | string | BigNumber;
   max?: number | string | BigNumber;
   type?: string;
   toDecimals?: number;
 }
 
-export const convertToOdds = ({
+export const convertToOdds = (normalizedPrice, toDecimals = 4) => {
+  const { oddsType } = useAppStatusStore();
```

Comment:
this shouldn't work since this isn't a react component. I believe you have to use `AppStatus.get()` in this case.